txtorcon/torinfo.py

```diff
@@ -99,6 +109,9 @@ def stripper(key, arg):
 
         d = self.proto.get_info_raw(req)
         d.addCallback(functools.partial(stripper, req))
+        descriptor_keys = ["desc/name", "desc/id"]
+        if self.info_key in descriptor_keys and self.proto.use_stem:
+            d.addCallback(_parse_stem_descriptor)
         return d
```

Comment:
Updated in the commit https://github.com/sammyshj/txtorcon/commit/dc6650c46e37864575b7f7e305dfdc97a2d5a1ea
Should I squash all the commits into one now?